### ISSUE TITLE
MoneroKon 2022 Planning Meeting: Saturday 21st May 2022 @ 18:00 UTC

### ISSUE THREAD

**Issue by ajs-xmr** [monero-project/meta#705]
**Location **

- IRC: #monero-events
- Matrix: [#monero-events:monero.social](https://matrix.to/#/#monero-events:monero.social?via=matrix.org&via=monero.social) ([Instructions for joining the Matrix server](https://forum.monero.space/d/79-how-to-join-the-monero-core-team-matrix-server-web))
- Telegram: https://t.me/monerokon

**Time**: 18:00 UTC

**Agenda**:
- Greetings
- Invoices for community approval (DR)
- Update on venue requirements q list (G+H)
- CfP status (ajs/all)
- Status of merch delivery (msvb)
- Ticket sales target reached. Should we increase the supply? (ajs/all)
- Banners (ajs)
- Food-trucks (G+H)
- Night-time programming (midipoet/G+H)
- Venue mural [inside] (venue/G+H)
- AOB
- Confirm next meeting time and date

Please comment on GitHub in advance of the meeting if you would like to propose an agenda item.

**Comment by umma08** [comment]
Logs below

```
[2022-05-21 19:00:14] <midipoet> It is MoneroKon 2022 planning meeting time
[2022-05-21 19:00:17] <midipoet> Agenda is here: https://github.com/monero-project/meta/issues/705
[2022-05-21 19:00:20] <midipoet> 1. Greetings
[2022-05-21 19:00:47] <monerokon-tg> [telegram] <franciscom0301>Hello all
[2022-05-21 19:01:33] <msvb-web> Hello.
[2022-05-21 19:02:23] <anhdres[m]> Hi there 
[2022-05-21 19:02:26] <midipoet> 2. Invoices for community approval (DR)
[2022-05-21 19:02:40] <midipoet> we have quite a few to get through this week for vote
[2022-05-21 19:03:04] <ajs_[m]> hi
[2022-05-21 19:03:19] <midipoet> The first is the Arroz Studies n/t invoice: https://cryptpad.fr/file/#/2/file/efuJs7f7V6ZeBglpvsbbhXbJ/
[2022-05-21 19:03:43] <midipoet> we are still waiting for a more detailed invoice, so this can wait (it should include artists, equipment, staffing, etc)
[2022-05-21 19:04:23] <midipoet> the second one is for Van rental (so we can move equipment about for the event): https://cryptpad.disroot.org/file/#/2/file/+hgeq-66XFQoUqNhrFT9J42M/
[2022-05-21 19:04:44] <midipoet> the van is booked in my name and is for 4 days
[2022-05-21 19:04:45] <ajs_[m]> is this one automatic?
[2022-05-21 19:05:05] <midipoet> ajs_[m]: i don' think so. 
[2022-05-21 19:05:25] <midipoet> i wasn't given that option. only had an option for size of van (as far as i could tell)
[2022-05-21 19:06:44] <midipoet> please note that i have paid for this rental, so the reimbursement would be to me
[2022-05-21 19:07:09] <monerokon-tg> [telegram] <franciscom0301>Do we have space to park the van on the venue? (re @xmr_events_bot: (irc) midipoet the van is booked in my name and is for 4 days)
[2022-05-21 19:07:56] <msvb-web> Seems okay.
[2022-05-21 19:07:57] <midipoet> @frasciscom0301 i have not enquired about that, but it is doubtful, so may have to park off site
[2022-05-21 19:08:13] <midipoet> can we vote: Approve (+1) Disapprove (-1) or Abstain (0)
[2022-05-21 19:09:51] <ajs_[m]> +1
[2022-05-21 19:10:14] <monerokon-tg> [telegram] <franciscom0301>+1
[2022-05-21 19:11:42] <midipoet> msvb-web: just your vote we are waiting on
[2022-05-21 19:12:58] <msvb-web> Approve.+1
[2022-05-21 19:13:10] <midipoet> cool. 
[2022-05-21 19:13:26] <msvb-web> Sorry, in a conference right now a real one I mean.
[2022-05-21 19:14:06] <midipoet> ok, next one is also from myself (reimbursement for volunteer expenses +flights+travel). it was agreed to invoice these now, as they are starting to get paid on credit cards. 
[2022-05-21 19:14:15] <midipoet> https://cryptpad.disroot.org/file/#/2/file/ldDBzdUtOVPYV5SatgtmQc4J/
[2022-05-21 19:15:04] <midipoet> i will abstain from the vote, but same as last time: Approve (+1) Disapprove (-1) or Abstain (0)
[2022-05-21 19:15:13] <monerokon-tg> [telegram] <franciscom0301>Ok, forgot to tell but I won't demand any reimbursement as volunteer, for obvious reasons (re @xmr_events_bot: (irc) midipoet ok, next one is also from myself (reimbursement for volunteer expenses +flights+travel). it was agreed to invoice these now, as they are starting to get paid on credit cards.)
[2022-05-21 19:16:02] <monerokon-tg> [telegram] <franciscom0301>Abstain also
[2022-05-21 19:16:15] <monerokon-tg> [telegram] <franciscom0301>0
[2022-05-21 19:16:37] <ajs_[m]> +1
[2022-05-21 19:16:45] <msvb-web> Approve +1.
[2022-05-21 19:17:16] <midipoet> ok. thanks. 
[2022-05-21 19:17:39] <midipoet> next up is an invoice from AJ (fro volunteer travel expenses)
[2022-05-21 19:17:41] <midipoet> https://cryptpad.fr/file/#/2/file/YCKcYxRRZh5s89EvTEj9NeTV/
[2022-05-21 19:18:00] <midipoet> so vote is: Approve (+1) Disapprove (-1) or Abstain (0)
[2022-05-21 19:18:05] <midipoet> my vote is +1
[2022-05-21 19:18:20] <ajs_[m]> abstain
[2022-05-21 19:19:06] <monerokon-tg> [telegram] <franciscom0301>+1
[2022-05-21 19:19:26] <msvb-web> Approve +1.
[2022-05-21 19:19:42] <midipoet> ok great. 
[2022-05-21 19:20:02] → anticarnist joined (~user@2001:818:d9b0:4f00:9e94:aede:821e:3c1a)
[2022-05-21 19:20:04] ⇐ mfoolb quit (~mfoolb@user/mfoolb): Ping timeout: 240 seconds
[2022-05-21 19:20:06] <midipoet> the final one is from MoneroTalk. 
[2022-05-21 19:21:03] <midipoet> it was agreed that both Doug and Sunita would be of great benefit to be present at MoneroKon. they are going to be volunteers, help with the MC/hosting, do their MoneroTalk thing, and also have their Gratuitas Coffee stall
[2022-05-21 19:21:31] <midipoet> as they are coming stateside, their flights were not cheap though (unfortunately). 
[2022-05-21 19:21:33] <midipoet> https://cryptpad.disroot.org/file/#/2/file/WIYFKOesa-0FFDXQ3DsHsOJ4/
[2022-05-21 19:22:12] <midipoet> at the sub-meeting we had discussed a €1000 limit for volunteer expenses, unless there were mitigating circumstances. 
[2022-05-21 19:22:31] <monerokon-tg> [telegram] <franciscom0301>1000 seems reasonable
[2022-05-21 19:24:00] <midipoet> in the case of MoneroTalk, their flights will be €1250 each. It is not ideal, but - personally i think worth it. Doug and Sunita and Monerotalk are an important presence, and they have helped out at numerous Monero events in the past (so are trusted volunteers for all sorts of help that will be required). 
[2022-05-21 19:24:47] → mfoolb joined (~mfoolb@user/mfoolb)
[2022-05-21 19:24:59] <monerokon-tg> [telegram] <franciscom0301>Well, they deserve for the contribution they will do (re @xmr_events_bot: (irc) midipoet in the case of MoneroTalk, their flights will be €1250 each. It is not ideal, but - personally i think worth it. Doug and Sunita and Monerotalk are an important presence, and they have helped out at numerous Monero events in the past (so are trusted volunteers for all sorts of help that will be required).)
[2022-05-21 19:25:08] <midipoet> so unless there are any questions, comments, concerns on this, the vote is: Approve (+1) Disapprove (-1) or Abstain (0)
[2022-05-21 19:25:15] <midipoet> my vote is +1
[2022-05-21 19:25:34] <monerokon-tg> [telegram] <franciscom0301>+1
[2022-05-21 19:26:05] <ajs_[m]> +1
[2022-05-21 19:26:31] <msvb-web> +1.
[2022-05-21 19:27:28] <midipoet> ok. thanks all. i will move these to pending payment, and ask DR to process
[2022-05-21 19:27:47] <midipoet> 3. Update on venue requirements q list (G+H)
[2022-05-21 19:28:11] <midipoet> the latest version of the doc is here: https://cryptpad.disroot.org/pad/#/2/pad/edit/NnjBlAV78oEKScZKVnqMOtox/
[2022-05-21 19:28:23] <midipoet> it has been updated with some new questions, and suggestions.
[2022-05-21 19:29:08] <midipoet> are there any comments questions on any topic?
[2022-05-21 19:29:14] <midipoet> *comments or questions
[2022-05-21 19:30:29] <anhdres[m]> I do have a question regarding the mural
[2022-05-21 19:31:18] <midipoet> go for it
[2022-05-21 19:32:49] <anhdres[m]> I don't know how big the wall is, or even if there's one at all, the other day msvb-web suggested I might paint one if desired. I don't want to step on anyone's toes, but if there's interest I can do it. I guess I'd have to but the materials over there, and arrive at least a day before, but I can do it.
[2022-05-21 19:33:37] <anhdres[m]> regarding style, I could do a quick mockup tomorrow for example, so you could see style
[2022-05-21 19:34:02] <midipoet> anhdres[m]: from my perspective, you are the first artist to actually say they will do it. we have tried to contact a few, and not really had much confirmation or contact from them
[2022-05-21 19:34:13] <monerokon-tg> [telegram] <franciscom0301>Regarding mural I could assume we won't need more artists for the wall, right?
[2022-05-21 19:34:34] ⇐ mfoolb quit (~mfoolb@user/mfoolb): Ping timeout: 240 seconds
[2022-05-21 19:34:58] <msvb-web> anhdres[m] should do it of course, but it's quite inconvenient for him. I'm so happy he wants to help.
[2022-05-21 19:35:28] <ajs_[m]> if we can, i would like to construct a wall to use as background on the main stage
[2022-05-21 19:35:42] <midipoet> anhdres[m]: i think do a mock-up if its not too much trouble, and then we can plan that way. will the day before be enough time?
[2022-05-21 19:35:46] <anhdres[m]> ajs_[m]: that would be awesome
[2022-05-21 19:35:52] <msvb-web> Worst case we have two people painting two walls, which could be great fun.
[2022-05-21 19:35:55] <ajs_[m]> and have anhdres go at it
[2022-05-21 19:36:19] <monerokon-tg> [telegram] <franciscom0301>That would be perfect anhdres
[2022-05-21 19:36:30] <anhdres[m]> most here know some of the monero stuff I've already done, most are vector, but I can paint with brushes and shouldn't be much worse
[2022-05-21 19:36:47] <monerokon-tg> [telegram] <franciscom0301>Let us know if we could help somehow to facilitate your job
[2022-05-21 19:36:55] <anhdres[m]> maybe something inspiration like this one but simpler (for time) 
[2022-05-21 19:36:57] — anhdres[m] uploaded an image: (273KiB) < https://libera.ems.host/_matrix/media/r0/download/matrix.org/CAVyCwxIsNdqYhsOoiyryZpt/image.png >
[2022-05-21 19:37:12] <monerokon-tg> [telegram] <franciscom0301>Really nice (re @xmr_events_bot: (irc) anhdres[m] uploaded an image: (273KiB) < https://libera.ems.host/_matrix/media/r0/download/matrix.org/CAVyCwxIsNdqYhsOoiyryZpt/image.png >)
[2022-05-21 19:37:13] <midipoet> sounds cool. the only thing is that the existing wall of the venue may need to be whitewashed (and dry) before being painted over (as i understand wall art)
[2022-05-21 19:37:30] <midipoet> but if we are building a wall, that is less of a problem
[2022-05-21 19:37:36] <anhdres[m]> midipoet: yes, that's why I think two days are needed
[2022-05-21 19:38:02] <midipoet> anhdres[m]: the only thing we have to check is whether you can get access to the venue before set-up day
[2022-05-21 19:38:16] <anhdres[m]> I still have no flights or anything, that's why I wanted to know about this so I can try to get earlier
[2022-05-21 19:38:21] <anhdres[m]> yep
[2022-05-21 19:38:48] <anhdres[m]> otherwise you're forced to paint it beforehand off location (on that wood wall) and then assemble at the venue
[2022-05-21 19:39:32] <midipoet> ok. but do we want to keep the resident artist as well, or not? 
[2022-05-21 19:39:39] <midipoet> (question for all)
[2022-05-21 19:39:47] <ajs_[m]> nah
[2022-05-21 19:39:53] <anhdres[m]> I've checked the photos of the venue and I could try to match the graphic style they use, looks fun
[2022-05-21 19:40:08] → mfoolb joined (~mfoolb@user/mfoolb)
[2022-05-21 19:41:29] <anhdres[m]> midipoet: setup day is the 17th right?
[2022-05-21 19:41:35] <midipoet> yes
[2022-05-21 19:42:13] <midipoet> ok. so anhdres[m] please do a mock up, and we will confirm on Wednesday. would that be ok?
[2022-05-21 19:42:22] <anhdres[m]> yes
[2022-05-21 19:42:25] <midipoet> great
[2022-05-21 19:42:42] <anhdres[m]> I'd assume a 4:3 aspect ratio of sorts
[2022-05-21 19:42:43] <midipoet> ok, so unless any other questions
[2022-05-21 19:42:44] <ajs_[m]> anhdres: if we build a wall for the main stage, size should be around 4 meters x 2 meters
[2022-05-21 19:42:46] <midipoet> 4. CfP status (ajs/all)
[2022-05-21 19:43:11] <midipoet> ajs_[m]: could we not just as easily buy a plain white banner/backdrop
[2022-05-21 19:43:20] <midipoet> that we could paint on?
[2022-05-21 19:43:47] <ajs_[m]> I'll look into it
[2022-05-21 19:45:32] <anhdres[m]> as a rule of thumb, the firmer the surface, the faster (and steady) I can paint it
[2022-05-21 19:45:53] <midipoet> anhdres[m]: good point!
[2022-05-21 19:47:05] <midipoet> ajs_[m]: do you want to provide a schedule/CfP update?
[2022-05-21 19:47:51] <ajs_[m]> I've email all speakers and waiting on confirmations
[2022-05-21 19:48:13] <ajs_[m]> 10 have responded 
[2022-05-21 19:48:40] <ajs_[m]> I set a deadline for Wednesday before the mid-week meeting
[2022-05-21 19:49:12] <ajs_[m]> current schedule: https://monerokon.com/pages/schedule 
[2022-05-21 19:50:01] <ajs_[m]> I am updating the schedule as confirmations come in
[2022-05-21 19:50:04] ⇐ mfoolb quit (~mfoolb@user/mfoolb): Ping timeout: 240 seconds
[2022-05-21 19:50:34] <monerokon-tg> [telegram] <franciscom0301>Great, thanks ajs 👍
[2022-05-21 19:50:41] <midipoet> great. thanks
[2022-05-21 19:50:49] <ajs_[m]> some that have been offered a lightning talk, asked if they could be allocated more time (at least 20 minutes)
[2022-05-21 19:51:26] <ajs_[m]> I am not sure what to say to them
[2022-05-21 19:51:29] <midipoet> i, personally, don't think we should start those sorts of convos (as it would get messy quickly)
[2022-05-21 19:52:37] <monerokon-tg> [telegram] <franciscom0301>Yeah, I think it may make hard for us to extend those lightning talks
[2022-05-21 19:53:08] <monerokon-tg> [telegram] <franciscom0301>They could always discuss their work with the ones interested after the talks
[2022-05-21 19:53:28] <monerokon-tg> [telegram] <franciscom0301>Yeah, I think it may make hard for our planning to extend those lightning talks
[2022-05-21 19:54:39] <ajs_[m]> okay, I'll try to diplomatic about it and kindly say sorry, but no
[2022-05-21 19:54:56] <midipoet> ajs_[m]: perhaps a polite response saying that the schedule is already full, and extending the days is not possible with the venue, due to contracted time for event per day
[2022-05-21 19:55:03] <ajs_[m]> that all on CfP
[2022-05-21 19:55:22] <midipoet> 5. Status of merch delivery (msvb)
[2022-05-21 19:55:51] <msvb-web> No report because I'm on mobile at a conference.
[2022-05-21 19:56:04] <msvb-web> Merchandise production is on time.
[2022-05-21 19:57:24] <midipoet> great
[2022-05-21 19:57:36] <midipoet> 6. Ticket sales target reached. Should we increase the supply? (ajs/all)
[2022-05-21 19:58:12] <ArticMine[m]> If possible I say yes
[2022-05-21 19:58:43] <midipoet> ArticMine[m]: it's not as easy, unfortunately. the risk is rain
[2022-05-21 19:58:49] <msvb-web> Yes!
[2022-05-21 19:58:55] <midipoet> and what we would have to do, if it rained, and we had to move inside
[2022-05-21 19:58:55] <monerokon-tg> [telegram] <franciscom0301>No opinion on that matter, will trust in your ideas
[2022-05-21 19:59:47] <monerokon-tg> [telegram] <franciscom0301>It rarely rains in June however
[2022-05-21 20:00:26] <monerokon-tg> [telegram] <franciscom0301>It rarely rains in June, but we could have bad luck off course
[2022-05-21 20:01:28] <ajs_[m]> https://www.holiday-weather.com/lisbon/averages/june/
[2022-05-21 20:01:32] <ArticMine[m]> Can we have the latecomers accept the rain risks
[2022-05-21 20:01:55] → mfoolb joined (~mfoolb@user/mfoolb)
[2022-05-21 20:02:02] <ArticMine[m]> And stand in the rain
[2022-05-21 20:02:10] <midipoet> https://usercontent.irccloud-cdn.com/file/EgRuQTrc/Screenshot%202022-05-21%20at%2020.02.03.png
[2022-05-21 20:02:11] <midipoet> LOL
[2022-05-21 20:02:27] <midipoet> that was 2021
[2022-05-21 20:03:05] <monerokon-tg> [telegram] <franciscom0301>Yeah, weather is kinda messy currently (re @xmr_events_bot: (irc) midipoet https://usercontent.irccloud-cdn.com/file/EgRuQTrc/Screenshot%202022-05-21%20at%2020.02.03.png)
[2022-05-21 20:03:22] <midipoet> so extra/excess tickets sold, don't allow inside entry?
[2022-05-21 20:03:26] <midipoet> seems unethical
[2022-05-21 20:03:50] <midipoet> i guess the risk is that we have limited inside seating if it rains
[2022-05-21 20:04:18] <ajs_[m]> it is not too bad of a risk
[2022-05-21 20:04:42] <midipoet> ok. so how many more tickets are we proposing to offer?
[2022-05-21 20:05:23] <ajs_[m]> i am more concerned with the lack of volunteers
[2022-05-21 20:06:25] <midipoet> we could offer volunteer tickets
[2022-05-21 20:06:41] <ajs_[m]> venue says max capacity is 400 
[2022-05-21 20:07:13] <ajs_[m]> with current sales and speakers, we are at 150-170
[2022-05-21 20:07:23] <midipoet> i was under the impression that was total capacity (inside+outside)
[2022-05-21 20:07:39] <midipoet> or inside, but not seated theatre style
[2022-05-21 20:08:35] <ajs_[m]> and they have their own members coming too
[2022-05-21 20:08:44] <msvb-web> Difficult question, rain is the only good reason to limit.
[2022-05-21 20:09:20] <anhdres[m]> release 50 extra bring-your-own-umbrella tickets
[2022-05-21 20:09:27] <ajs_[m]> so say about 100 for members and venue staff
[2022-05-21 20:09:53] <msvb-web> There are lots of camp events like Honey Badger that are exclusively outdoors with no rain plan.
[2022-05-21 20:10:02] <midipoet> okay. lets sell tickets in limited batches of 10, and see how we go over the next few days (before Wed and then next Saturday)?
[2022-05-21 20:10:07] <anhdres[m]> now seriously, probably volunteer tickets should get a couple of people already living in europe interested
[2022-05-21 20:10:40] <midipoet> though, i do like the idea of volunteer tickets. 
[2022-05-21 20:10:57] <midipoet> ie. extra tickets come with a caveat that you have to help
[2022-05-21 20:12:04] ⇐ mfoolb quit (~mfoolb@user/mfoolb): Ping timeout: 240 seconds
[2022-05-21 20:12:25] <msvb-web> So should we call the new tickets special, name them volunteer helper tickets? Or do we KISS and call them general entrance tickets?
[2022-05-21 20:12:52] <msvb-web> KISS means keep it simple stupid.
[2022-05-21 20:12:54] <ajs_[m]> it is going to be hard to enforce though... are we going to bounce people for being slackers
[2022-05-21 20:13:02] <msvb-web> I'm not going to kiss you
[2022-05-21 20:13:18] <monerokon-tg> [telegram] <franciscom0301>Hahahahaha
[2022-05-21 20:13:50] <midipoet> okay. i don' think we are going to resolve this today
[2022-05-21 20:14:00] <midipoet> so let's move it to Wednesday meeting
[2022-05-21 20:14:01] <msvb-web> Wednesday.
[2022-05-21 20:14:14] <midipoet> 7. Banners (ajs)
[2022-05-21 20:14:16] <ajs_[m]> i say just open 50 GA and call it a dau
[2022-05-21 20:14:17] <ajs_[m]> day
[2022-05-21 20:14:42] <ArticMine[m]> We can say the early ones have priority for indoor seating
[2022-05-21 20:14:53] <midipoet> ajs_[m]: yeah, that sort of makes sense - but let's discuss on Wednesday. we still have time. 
[2022-05-21 20:15:05] <ajs_[m]> sponsor banners are done and delivered
[2022-05-21 20:15:08] <ArticMine[m]> But yes KISS
[2022-05-21 20:15:21] <midipoet> ajs_[m]: cool. MoneroTalk were wondering could we get a banner for them?
[2022-05-21 20:15:28] <midipoet> is that still possible?
[2022-05-21 20:15:37] <midipoet> they have provided a graphic
[2022-05-21 20:15:56] <ajs_[m]> yeah, supplier has fast turn around 
[2022-05-21 20:16:04] <midipoet> ok great. will DM you
[2022-05-21 20:16:09] <ajs_[m]> k
[2022-05-21 20:16:27] <midipoet> we have gone over - so very quickly items 8-10 are all WIP
[2022-05-21 20:16:33] <midipoet> and on the venue requirements doc
[2022-05-21 20:16:42] <anhdres[m]> are banners just for sponsors or for general community projects?
[2022-05-21 20:16:43] <midipoet> so 
[2022-05-21 20:16:47] <midipoet> 11. AOB?
[2022-05-21 20:16:58] <midipoet> (any other business)
[2022-05-21 20:17:37] <ajs_[m]> anhdres:  i printed for sponsors, but if you want to have something printed, we could get it ordered
[2022-05-21 20:17:53] <midipoet> ok. 12. Confirm next meeting time and date
[2022-05-21 20:17:56] <midipoet> same time next week
[2022-05-21 20:18:01] <midipoet> (sorry for going over time)
[2022-05-21 20:18:14] <monerokon-tg> [telegram] <franciscom0301>Who were dealing with speakers' dinner? Let me know if you need any help to clarify our requirements/demands in Portuguese
[2022-05-21 20:18:16] <anhdres[m]> ajs_[m]: ok will think about it, thanks
[2022-05-21 20:19:16] → mfoolb joined (~mfoolb@user/mfoolb)
[2022-05-21 20:19:23] <midipoet> @franscisom0301
[2022-05-21 20:19:41] <midipoet> we will know this week. i think G+H are at the venue on Monday and see how they go
[2022-05-21 20:20:05] <midipoet> we may ask you to call and confirm all details/clarifications if they have some translation concerns
[2022-05-21 20:20:54] <monerokon-tg> [telegram] <franciscom0301>Ok, perfect then (re @xmr_events_bot: (irc) midipoet we may ask you to call and confirm all details/clarifications if they have some translation concerns)
[2022-05-21 20:20:56] <monerokon-tg> [telegram] <franciscom0301>Thanks
[2022-05-21 20:20:57] <ajs_[m]> by they way, live stream is all set up
[2022-05-21 20:21:16] <midipoet> ajs_[m]: very cool!
[2022-05-21 20:21:20] <ajs_[m]> https://live.monerokon.com
[2022-05-21 20:21:43] <ajs_[m]> right now I am streaming xmr.radio for testing
[2022-05-21 20:24:37] <msvb-web> Good meeting and great moderation midipoet, dankon.
```
